console/src/components/Services/Data/TableCommon/TableHeader.js

```diff
@@ -19,6 +19,7 @@ import {
   getTablePermissionsRoute,
   getTableRelationshipsRoute,
 } from '../../../Common/utils/routesUtils';
+import {getReadableNumber} from "../../../Common/utils/jsUtils";
```

Comment:
I see some inconsistent formatting in some places, for example, here. But not like badly formatted, but rather not formatted at all. In another place, there was `set:{` — no space between `:` and `{`.

 Is it possible that you disabled git hook?

Reply:
Nope. My hook is active. This is odd.